bson/src/test/unit/org/bson/codecs/pojo/PojoRoundTripTest.java

```diff
@@ -507,6 +508,10 @@ public void test() {
                 getPojoCodecProviderBuilder(MapListGenericExtendedModel.class),
                 "{values: {a: [1, 2, 3], b: [4, 5, 6]}}"));
 
+        data.add(new TestData("BsonRepresentation is encoded and decoded correctly", new BsonRepresentationModel(1),
```

Comment:
Also add a roundtrip test without the default conventions applied.

Reply:
Done, but I had to add it to `PojoCustomTest`, because `PojoRoundTripTest` does tests with the "AUTOMATIC_BUILDER" and "PACKAGE_BUILDER" which both use default conventions.